values.yaml

```diff
@@ -169,6 +169,15 @@ mastodon:
     # requests:
     #   cpu: 250m
     #   memory: 128Mi
+    # -- (Streaming Deployment) liveness and readiness probes
+    livenessProbe:
+      httpGet:
+        path: /api/v1/streaming/health
+        port: streaming
+    readinessProbe:
+      httpGet:
+        path: /api/v1/streaming/health
+        port: streaming
```

Comment:
In best case we set `failureThreshold` and `periodSeconds` already explicitly for each probe, taking away potential assumptions.

Reply:
You are making me responsible for something which is not already set today... but I can do that easily enough.

Reply:
We should have them for all probes, including readiness and liveness probes.

And for the startup probe we might want to reduce the interval to 3 seconds and increase the failure theshold to 10 or even 20.

This will provide the container with quicker startup times, since the kubelet will retry every 3 seconds instead of every 10, and the increased theshold allows startups that range from 30-60 seconds (depensing on the usage of 10 or 20).

Reply:
You keep adding things to your laundry list of desires.  Can you limit your scope creep here, decide a final answer of what you want, such that I can just do it and you can approve it?

I get you're trying to improve things, but you've moved the goalposts twice on me now.

Reply:
New commit up.  Please advise if that is all you will make me improve on top the intent of the PR I filed, or if I need to do more things.

Reply:
the failureThresholds for the liveness and readiness probes are way too high.

But don't worry, I can also just provide a PR to fix that later, if you don't want to bother. :)

Reply:
done.